README.md

```diff
@@ -2040,6 +2041,14 @@ The following text-specific constant options are also supported:
 
 If a **lineWidth** is specified, input text values will be wrapped as needed to fit while preserving existing newlines. The line wrapping implementation is rudimentary; for non-ASCII, non-U.S. English text, or for when a different font is used, you may get better results by hard-wrapping the text yourself (by supplying newlines in the input). If the **monospace** option is truthy, the default **fontFamily** changes to “ui-monospace, monospace”, and the **lineWidth** option is interpreted as characters (ch) rather than ems.
 
+If the **textOverflow** option is specified, only one line will be shown: if the text is longer than the given line width, it is clipped to that length, a title with the complete text is provided unless the mark already has a title channel. The following textOverflow values are supported:
```

Comment:
This statement makes me wonder if the **textOverflow** behavior should apply after splitting into lines (after wrapping, potentially, or given hard-wrapped input with newlines already present), and that we should need have a separate **whiteSpace** option to control whether or not we wrap (independently of overflow). We support a mix of _pre_, _pre-wrap_, and _nowrap_ currently.

Reply:
I’ve now structured the code this way, such that **textOverflow** can now apply to multiple lines if you insert newlines manually. We’d still need to add a **whiteSpace** option to unlock more of this functionality (and maybe another option to specify the maximum number of lines). I don’t think we should add any more functionality to this PR, though.